.chloggen/TA-update-configs-to-enable-mtls.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: target allocator collector
```

Comment:
I think this should be only `target allocator`

Reply:
My thought is that since we are modifying the collector's configuration applied by the operator, it should be mentioned.

Reply:
or use a list: `target allocator, collector`

Reply:
there is missing `,`